### DISCUSSION TITLE
Not able to receive forgot password email

### DISCUSSION THREAD

**Discussion by pentester303** [community/community#142121] [Other Feature Feedback, Questions, & Ideas]
Hello Team,

I am not able to reset my password as I am not able to receive the forgot password email. I tried several times

**Comment by jheisonnovak** [comment]
Hi there,

If you're not receiving the password reset email, here are a few things you can try:

Check Spam/Junk Folder: Sometimes, the reset email can end up in your spam or junk folder, so it's worth checking there.
Email Address: Make sure you're entering the correct email address associated with your GitHub account.
Email Filters: Check if you have any email filters or rules that might be redirecting or blocking emails from GitHub.
Email Provider Delays: Some email providers might take longer to process the email. It might be worth waiting a few minutes and trying again.
Contact GitHub Support: If you've tried the above steps and still aren't receiving the email, you may need to reach out directly to [GitHub Support](https://support.github.com/). They can help you troubleshoot the issue further.
Hope this helps, and good luck with getting your password reset!

**Comment by pentester303** [comment]
Hi there

I have done all of these and even reached github support but still no
response.

**Comment by pentester303** [comment]
Hi

I created another account and asked for the forgot password link, I received it immediately however I did not receive it on my account. My username on https://www.npmjs.com/ is pentester303.

**Comment by jheisonnovak** [comment]
In this case, you can contact NPM support using: https://www.npmjs.com/support. Remember to check your email spam folder.

**Comment by pentester303** [comment]
I am not able to get support I already did contact support
